test/dns_nsupdate/dns_nsupdate_test.rb

```diff
@@ -1,8 +1,6 @@
 require 'test_helper'
-require 'dns/dns'
-require 'dns_nsupdate/dns_nsupdate_plugin'
+require 'dns_common/dns_common'
 require 'dns_nsupdate/dns_nsupdate_main'
-require 'dns_nsupdate/dns_nsupdate_gss_main'
```

Comment:
I believe https://github.com/theforeman/smart-proxy/pull/492/commits/cb8112e021421f699183c4d23162ac89dd184367 fixes it properly.